extensions/blocks/simple-payments/editor.scss

```diff
@@ -44,6 +49,7 @@
 			line-height: 27px;
 			margin-bottom: 0;
 			font-weight: 400;
+			display: block;
 		}
 
 		.components-select-control__input,
```

Comment:
I get a one pixel margin around the select input, so maybe add `margin: 0` here?

<img width="218" alt="image" src="https://user-images.githubusercontent.com/96462/97573943-66077480-19e2-11eb-8725-e1d3adf8bda8.png">

Reply:
nice 👁️ I think it's addressed here 6fb645026162ec22c002e1f9b219f68817ccf77e

Reply:
That's not fixed it for me, unfortunately. The margin is around the select with class `components-select-control__input`. It looks like that change removes the margin from the labels.

Reply:
it should be applied to the input :-/

![image](https://user-images.githubusercontent.com/77539/97594914-eda0b380-19e1-11eb-9868-71f66eca15e5.png)

Reply:
I think it's working now (screenshot from self-hosted):

<img width="206" alt="Screen Shot 2020-10-29 at 3 06 31 PM" src="https://user-images.githubusercontent.com/1689238/97621087-cf927d80-19f8-11eb-8fc3-235b14518841.png">